tobac/tracking.py

```diff
@@ -304,10 +384,7 @@ def linking_trackpy(
     ):
         cell = int(i_particle + cell_number_start)
         particle_num_to_cell_num[particle] = int(cell)
-
-    remap_particle_to_cell_vec = np.vectorize(
-        lambda particle_cell_map, input_particle: particle_cell_map[input_particle]
-    )
+    remap_particle_to_cell_vec = np.vectorize(remap_particle_to_cell_nv)
```

Comment:
Yeah, the particle numbers are not necessarily contiguous, so this doesn't work if we want contiguous cell numbers. My guess is that there's an easier solution here, but I don't think this code will work to provide contiguous cell numbers. 

It's worth noting that we don't guarantee contiguous cell numbers; in fact, cell numbers won't be contiguous in many cases, but that's a separate issue. This will at least result in cell numbers starting at ~cell_number_start and ~incrementing from there (the particle numbers can end up quite large).

Reply:
Ok I see, I did not think about this. Then we should go with this working solution as it is.